include/gnc_constants.hpp

```diff
@@ -0,0 +1,52 @@
+//
+// include/gnc_constants.hpp
+// PSim
+//
+// Contributors:
+//   Kyle Krol  kpk63@cornell.edu
+//
+// Pathfinder for Autonomous Navigation
+// Space Systems Design Studio
+// Cornell Univeristy
+//
+
+/* - NOTICE - Values in this file should be seldomly edited and have hidden
+ *            interdependancies. */
+
+#ifndef PAN_PSIM_INCLUDE_GNC_CONSTANTS_HPP_
+#define PAN_PSIM_INCLUDE_GNC_CONSTANTS_HPP_
+
+#include "lin.hpp"
+
+namespace gnc {
+namespace constant {
+
+/** Initial GPS week number that serves as the 'PAN epoch'. Some of the
+ *  constants below depend on this number. */
+constexpr static unsigned int init_gps_week_number = 2045;
+
+/** Earth's intertial rotation rate about the z axis in the ECEF frame.
+ *  NOTE: Pulled from PSim. */
```

Comment:
Is it worth adding more specific references here? Someone like me who doesn't know psim wouldn't know where to look

Reply:
Yeah no for sure that was the plan. This was just a very early mockup to make sure I wasn't putting time into something that we thought should be done another way.